receiver/memcachedreceiver/metadata.yaml

```diff
@@ -117,8 +117,12 @@ metrics:
       aggregation_temporality: cumulative
     attributes: [type,operation]
   memcached.operation_hit_ratio:
-    enabled: true
+    enabled: false
```

Comment:
We should leave this as `true` for at least one release so that users have a chance to adjust before changes are imposed on them.